lib/publish.js

```diff
@@ -34,7 +38,7 @@ const handleError = (task, err, tag, message) => {
 	return Observable.throw(err);
 };
 
-const publish = (task, tag) => Observable.fromPromise(npmPublish({tag}))
-	.catch(err => handleError(task, err, tag));
+const publish = (task, options) => Observable.fromPromise(npmPublish(options))
+	.catch(err => handleError(task, err, options));
```

Comment:
You have to change the `handleError` method. The third parameter is `options` now.

Reply:
thanks for that. I'm passing the tag, how it was before in the current code. So the behavior in that case is still the same

Reply:
No, you really need `options` entirely because `handleError` is invoking `publish` again so in case `contents` is set, that property is lost at the moment. So currently, if you pass in contents and it fails due to OTP, it asks for the OTP, tries again but the second time without contents property.

Reply:
Oh, I see. Good catch. Thanks for the feedback @SamVerschueren . Could take a look again, please?